sdk/pinocchio/src/sysvars/fees.rs

```diff
@@ -38,7 +38,10 @@ pub struct FeeRateGovernor {
     pub burn_percent: u8,
 }
 
+/// Default lamports per signature.
 pub const DEFAULT_TARGET_LAMPORTS_PER_SIGNATURE: u64 = 10_000;
+
+/// Default signatures per slot.
 pub const DEFAULT_TARGET_SIGNATURES_PER_SLOT: u64 = 50 * DEFAULT_MS_PER_SLOT;
 
 // Percentage of tx fees to burn...I think this was changed in the anza update...
```

Comment:
Looks this comment is also a bit strange, since the first one is `//` and the next line is `///` -- either way, the upcoming change rewards the full *priority* fee to validators, not base fee. And this is about base fee, so the burn percent is still valid